x-pack/plugins/observability/server/domain/services/validate_slo.ts

```diff
@@ -54,6 +58,12 @@ function validateSettings(slo: SLO) {
   }
 }
 
+function isValidId(id: string): boolean {
```

Comment:
Nice. Does `io-ts` not have runtime parsing / validating?

Reply:
Not as easy as doing `t.string(t.min(8), t.max(36))`. Instead you have to create a Type decoder, like we do for the Duration for example.